gtda/plots/homology.py

```diff
@@ -0,0 +1,85 @@
+"""Persistent homology-retated plotting functions """
+# License: GNU AGPLv3
+
+import numpy as np
+import plotly.graph_objs as gobj
+from gtda.diagrams._utils import _subdiagrams
+
+
+def plot_diagram(diagram, homology_dimensions=None, **input_layout):
+    """Plot a single persistence diagram.
+
+    Parameters
+    ----------
+    diagram : ndarray of shape (n_points, 3)
+        The persistence diagram to plot, where the third dimension along axis 1
+        contains homology dimensions, and the other two contain (birth, death)
+        pairs to be used as coordinates in the two-dimensional plot.
+
+    homology_dimensions : list of int or None, default: ``None``
+        Homology dimensions which will appear on the plot. If ``None``, all
+        homology dimensions which appear in `diagram` will be plotted.
+
+    """
+    if homology_dimensions is None:
+        homology_dimensions = np.unique(diagram[:, 2])
+
+    maximum_persistence = np.where(np.isinf(diagram), -np.inf, diagram).max()
```

Comment:
I think that the name is misleading as to what the variable contains.

Reply:
Renaming to `max_filt_param`, what do you think?

Reply:
sure, thanks